i18n/en.json

```diff
@@ -9493,7 +9493,7 @@
   },
   {
     "id": "web.error.unsupported_browser.min_os_version.mac",
-    "translation": "macOS 10.14+"
+    "translation": "macOS 10.15+"
```

Comment:
I don't even think Catalina is supported anymore, I think it's just Big Sur and up now:
https://endoflife.date/macos

Reply:
Updated to 11+.